ChangeLog

```diff
@@ -39,6 +39,7 @@
  * CMake >= 2.8.4 is required to build OpenTURNS
 
 === Bug fixes ===
+ * #948 (Bug with BayesDistribution)
```

Comment:
This can be backported to 1.11, could you move this changelog entry to 1.11 ?

Reply:
Done